Add docs for all-in-one image (#218)
kubernetes-sigs/kwok: site/content/en/docs/user/all-in-one-image.md

# Run cluster with all-in-one image

{{< hint "info" >}}

> This document walks you through how to create a cluster with all-in-one image.

Reviewer: ```suggestion
This document walks through the steps to create a cluster with the all-in-one image.
```
